File: 2.0/java/webhooks/getWebhookResource.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.webhooks.WebhooksApi;
import com.callfire.api.client.api.webhooks.model.ResourceType;
import com.callfire.api.client.api.webhooks.model.WebhookResource;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        WebhooksApi api = client.webhooksApi();
        WebhookResource resource = api.findWebhookResource(ResourceType.CALL_BROADCAST, "resource");
    }
}

File: 2.0/csharp/webhooks/getWebhookResource.cs
using CallfireApiClient;
using CallfireApiClient.Api.Webhooks.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var resource = client.WebhooksApi.FindWebhookResource(ResourceType.CALL_BROADCAST, "resource,events");
    }
}
